control-plane/plugin/src/resources/utils.rs

```diff
@@ -47,6 +47,19 @@ lazy_static! {
         "CHILD-STATUS-REASON",
         "REBUILD"
     ];
+    pub static ref REBUILD_HISTORY_HEADER: Row = row![
+        "CHILDURI",
+        "SRCURI",
+        "STATE",
+        "BLKTOTAL",
+        "BLKRECOVERED",
+        "BLKTRANSFERRED",
+        "BLKREMAINING",
```

Comment:
```suggestion
```
Don't think we need this since it's already completed

Reply:
I think it will be non zero for failed rebuild?

Reply:
Yes but we can infer that from total-recovered/transferred

Reply:
One can always use -oyaml to check all fields, but atm these formatted lines seem to be a bit too long for them to be useful

Reply:
Removed it